.circleci/config.yml

```diff
@@ -66,13 +64,12 @@ jobs:
           destination: raw-nightly-output
       - store_test_results:
           path: ~/project
-          destination: nightly-junit
+
   e2e:
     docker:
       - image: quay.io/influxdb/influxdb-circleci:latest
     environment:
       GOCACHE: /tmp/go-cache
-      GOFLAGS: "-mod=readonly -p=4" # Go on Circle thinks 32 CPUs are available, but there aren't.
```

Comment:
Bad copy-pasta, Go only gets confused about available CPUs in jobs that use `parallelism` > 1